### ISSUE TITLE
Flash of Unupdated Text in [Next Page ->]

### ISSUE THREAD

**Issue by tomByrer** [vuetifyjs/docs#2]
When clicking the bottom blue 'next page' button, the following occurs consistently.
1. Page jumps to header, with old / current webpage still in view.
2. New page / data suddenly gets displayed.
3. Repeatable, even if going back in history & re-clicking on [Next ->] (doesn't cache)
Seems unprofessional; reminds me of the AJAX days' of 'Flash of Unstylized Text'.  Suggestions:
* Use some sort of preloader to get a 'jump' on loading new data before the jump.
  + [vue-preload](https://github.com/egoist/vue-preload) may be helpful, but may not be compatible with Vue2.1.
* Wait until data is loaded before 'jumping'.  Might be an issue with the router?
* "Loading" notification may help (vue-preload has on built-in).  Or fade-out then quickly fade in when loaded.
* Cache data (maybe use a state management?  Not sure if [Vuex](https://github.com/vuejs/vuex) has history).

While this is a nit-pick of your attractive website, I assume the docs were built with your Vuetify at large?

**Comment by johnleider** [comment]
1. This is a before route call on the router to ensure the user is at the top of the page.
2. This should be a transition, if you are not seeing the page transition in and out, that is something I'll need to address.
3. HTML 5 history mode is on and I cannot repeat what you are describing.

To address your other issues, there are no data calls happening between views. Once the application loads it is done. I am running cache on all views within express using lru-cache.

The documentation was built fully with Vuetify and I would love to address your issues, but I'll need some additional testing steps, environments, or some example that really explains what you are having an issue with.

**Comment by tomByrer** [comment]
Firefox 53 & Chrome 55, Windows

Yes the page transition will make things look better, thank you.

**Comment by johnleider** [comment]
I completely agree! I'm just trying to nail down the specifics of what you are describing. Are you suggesting a preloader when the application initially loads, or between each view change? Thank you for the feedback.

**Comment by tomByrer** [comment]
The preloader I linked to has a trick: only preload when the mouse hovers over the link.  There is 150-300ms time delay between a hover & click.  With that JS, the next page is already loading before the user actually clicks on the anchor.
You could start to preload on a hover for the entire 'What's Next' section IMHO.

**Comment by tomByrer** [comment]
Perhaps the new VueJS 2.1.7 will fix it: https://github.com/vuejs/vue/issues/4510

**Comment by johnleider** [comment]
Thank you! I will get vuetify upgraded for vue@2.1.7 and will test.

**Comment by johnleider** [comment]
This should be resolved with the new Vue release. As I cannot reproduce this, if this is still continuing, open another issue please.
